simtools/db_handler.py

```diff
@@ -427,7 +427,7 @@ def readMongoDB(
 
         self._logger.debug("Trying the following query: {}".format(query))
         if onlyApplicable:
-            query["Applicable"] = onlyApplicable
+            query["Applicable"] = True
```

Comment:
Still interested why you can't write `query["Applicable"] = onlyApplicable`. Would that not work?

Reply:
Because then you will query either those which are False or those which are True. In our case we either want all parameters (i.e., not including that parameter in the query at all) or those which are True.